files/image_config/ntp/ntp-config.sh

```diff
@@ -29,4 +29,4 @@ get_database_reboot_type
 echo "Disabling NTP long jump for reboot type ${reboot_type} ..."
 modify_ntp_default "s/NTPD_OPTS='-g'/NTPD_OPTS='-x'/"
 
-systemctl restart ntp
+systemctl --no-block restart ntp
```

Comment:
explain the reason?

Reply:
ntp-config service should run and complete before starting the ntp service to make sure that the configs are updated before the daemon starts. This is achieved with "Before=ntp.service" on files ntp-config.service 

But files ntp-config.sh itself is trying to restarting ntp.service which bring in circular dependency. The --no-block helps in avoiding it.